### PR TITLE
Fix status and doc updates

### PR DESCRIPTION
**Pull request by ibirman-vbrick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +138,12 @@ vbrickEmbed.on('videoLoaded', data => {
 });
 ```
 
+Use the .off method to stop listening for events:
+
+```
+vbrickEmbed.off('videoLoaded');
```
Comment (lukeselden) [inline README.md:144]: This function needs a 2nd argument, which is the listener you added with `.on`

```js
const onVideoLoaded = data => {
		console.log(`Video loaded: ${JSON.stringify(data)}`);
};
vbrickEmbed.on('videoLoaded', onVideoLoaded);

// ....

vbrickEmbed.off('videoLoaded', onVideoLoaded);
```

File: README.md [hunk 2]
```diff
@@ -175,7 +181,23 @@ vbrickEmbed.setVolume(0.5);
...
+`vbrickEmbed.playerStatus;`
+
+#### Player Status Values
+* Buffering
+* Ended
+* Initializing
+* Paused
+* Playing
+* Seeking
+
```
Comment (lukeselden) [inline README.md:196]: Maybe also mention that the `playerStatusChanged` event reports when this value changes

File: README.md (hunk 1 above)
Comment (ibirman-vbrick) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ibirman-vbrick) [reply README.md]: done

File: demo/index.js [hunk 3]
```diff
@@ -59,8 +58,9 @@ function embedContent({
 	}));
 
 	function updateStatus() {
-		statusEl.innerHTML = currentEmbed.status || 'undefined';
-		playerStatusEl.innerHTML = currentEmbed.playerStatus || 'undefined';
+		statusEl.innerHTML = currentEmbed.isVod ?
```
Comment (lukeselden) [inline demo/index.js:61]: is this boolean inverted? if isVod === true then don't you want playerStatus?

Comment (ibirman-vbrick) [reply demo/index.js]: good point

File: README.md [hunk 4]
```diff
@@ -175,7 +181,23 @@ vbrickEmbed.setVolume(0.5);
...
 ```
 
-#### Removing Video
+##### Get Video Status
+You can get the status of the video using status variable. See example below
+
+`vbrickEmbed.playerStatus;`
+
+#### Player Status Values
+* Buffering
```
[the remark this answers is not in the document]
Comment (ibirman-vbrick) [reply README.md]: alphabetical?

**Review by lukeselden — APPROVED** [review]

**Pull request merged**